src/lib/y2storage/refinements/devicegraph_lists.rb

```diff
@@ -34,14 +34,25 @@ module DevicegraphLists
           disks:            DevicesLists::DisksList,
           partitions:       DevicesLists::PartitionsList,
           filesystems:      DevicesLists::FilesystemsList,
-          free_disk_spaces: DevicesLists::FreeDiskSpacesList
+          free_disk_spaces: DevicesLists::FreeDiskSpacesList,
+          lvm_vgs:          DevicesLists::LvmVgsList,
+          lvm_pvs:          DevicesLists::LvmPvsList,
+          lvm_lvs:          DevicesLists::LvmLvsList
+
         }
 
         DEVICE_LISTS.each do |list, klass|
           define_method(list) do
             klass.new(self)
           end
         end
+
+        alias_method :vgs, :lvm_vgs
+        alias_method :volume_groups, :lvm_vgs
+        alias_method :pvs, :lvm_pvs
+        alias_method :physical_volumes, :lvm_pvs
+        alias_method :lvs, :lvm_lvs
+        alias_method :logical_volumes, :lvm_lvs
```

Comment:
OK, it is matter of taste. But I still believe that if you see such lines, it is not obvious what it do unless you are very familiar with API:

```
vgs = devicegraph.volume_groups.with { |vg| vg.vg_name.starts_with?("system") }
vgs.lvs.with(lv_name: "swap")
vgs.lvm_lvs.with(lv_name: "root")
vgs.logical_volumes.with(lv_name: "boot")
```